android/BOINC/app/src/main/java/edu/berkeley/boinc/rpc/ProjectRelatedClasses.kt

```diff
@@ -51,22 +52,12 @@ data class ProjectConfig(
         parcel.readList(platforms.toList(), PlatformInfo::class.java.classLoader)
         termsOfUse = parcel.readString()
 
-        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.Q) {
-            val bArray = parcel.createBooleanArray()!!
-            usesName = bArray[0]
-            webStopped = bArray[1]
-            schedulerStopped = bArray[2]
-            accountCreationDisabled = bArray[3]
-            clientAccountCreationDisabled = bArray[4]
-            accountManager = bArray[5]
-        } else {
-            usesName = parcel.readBoolean()
-            webStopped = parcel.readBoolean()
-            schedulerStopped = parcel.readBoolean()
-            accountCreationDisabled = parcel.readBoolean()
-            clientAccountCreationDisabled = parcel.readBoolean()
-            accountCreationDisabled = parcel.readBoolean()
-        }
+        usesName = readBoolean(parcel)
```

Comment:
Is this version independent?
I remember you did similar changes in other classes that read/write from/to parcel.Could you please fix them too?